manifests/init.pp

```diff
@@ -15,6 +15,8 @@
 #
 # $proxy_password::     Proxy password for authentication
 #
+# $max_speed::          maximum bandwidth used per download in bytes/sec
```

Comment:
Also just realised: is this the download that pulp starts or the download speed clients can get?